RUNNING_LOCALLY.md

```diff
@@ -50,6 +50,24 @@ Change `register: country` to `register: school-type`.
 
 Update the schema in the same way, to `schema: school-type`.
 
+Update fields and registers location for environments different to beta in the same way.
```

Comment:
it is a very minor wording thing, but I would rather say _'other than'_ instead of _'different to'_